<ui version="4.0" >
 <class>ArticleSelectionDialog</class>
 <widget class="QDialog" name="ArticleSelectionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QTableWidget" name="tableWidget_articles" />
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="pushButton_ok" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_set" >
       <property name="text" >
        <string>&amp;Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_checkAll" >
       <property name="text" >
        <string>&amp;Check All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_uncheckAll" >
       <property name="text" >
        <string>&amp;Uncheck All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
